pkgs/applications/networking/cluster/gatekeeper/default.nix

```
@@ -2,17 +2,19 @@
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
+, gatekeeper
+, testers
```

```suggestion
, testers
```
Where is this dependency used?

---

I'd originally planned to add passthru tests however had abandoned this idea and forgot to remove the `gatekeeper` and `testers` dependencies. 

fixed in: https://github.com/NixOS/nixpkgs/pull/249129/commits/229a5a31687fa48ef91d6ac67e2ef4faf7d5d9f9